### PR TITLE
Canary v0.1.0

### PR DESCRIPTION
**Pull request by faultables**
In this PR I create Onboarding screen for new user.

@ri7nz can you please create HoC for validating authenticated user here?

### CODE REVIEW

**Comment by faultables** [comment]
Mana URL nya? @anakmagang

**Comment by anakmagang** [comment]
Current deployment URL: https://nadya-app-maa3x6a2c.now.sh

**Comment by faultables** [comment]
Sorry2 klo langsung push ke repo evilfactory, saya berjanji tidak akan mengulanginya lagi 🙏

**Comment by anakmagang** [comment]
Current deployment URL: https://nadya-app-hgfo0w14u.now.sh

File: app/src/fragments/Shell.js [hunk 1]
```diff
@@ -1,36 +1,64 @@
...
-function App() {
-  const classes = useStyles()
-  return (
-    <div className='App'>
-      <header className={classes.content}>
-        <p>I'm empty state</p>
-      </header>
-      <Link to='/pick'>
-        <Fab color='primary' aria-label='Add' className={classes.fab}>
+  componentDidMount() {
```
Comment (faultables) [inline app/src/fragments/Shell.js:25]: yang ini pak @ri7nz #23

**Pull request merged**